src/import-data/ImportDataOverview.ts

```diff
@@ -56,23 +53,13 @@ export const ImportDataOverview = (props: ImportDataOverviewProps): ReactNode =>
 
   return div({ style: styles.card }, [
     h2({ style: styles.title }, [getTitleForImportRequest(importRequest)]),
-    'url' in importRequest && div({ style: { fontSize: 16 } }, ['From: ', importRequest.url.hostname]),
+    h3({ style: { fontSize: 16 } }, ['Dataset source:']),
+    div({ style: { marginTop: '1rem' } }, [`${importRequest.url.href.split('?')[0]}`]),
```

Comment:
> hm not sure why the bee creation/build keep failing - everything seems to work for me locally ;/ will investigate

@yuliadub the build is failing because TypeScript is warning about this line. The `url` field does not exist on all import request types. That's why this line was conditional on `'url' in importRequest`.

Reply:
aha that makes sense, thanks for pointing that out!

Reply:
FYI, in case your editor doesn't show TypeScript errors inline, you can also run type checks locally with `yarn run tsc`.